File: Domain/TestResult.cs
using System;

namespace Domain
{
    public class TestResult
    {
        public Guid Id { get; set; }
        public Guid studentTestId { get; set; }
        public StudentTest studentTest { get; set; }
        public Guid testId {get;set;}
        public Test test { get; set; }
        public Guid answerId{get;set;}
        public Answer answer { get; set; } 
        public string Comment { get; set; }
    }
}

File: client-app/src/app/models/TestResult.ts
export interface ITestResult {
  id: string;
  studentTestId: string;
  testId: string;
  answerId: string;
  Comment: string;
}

export class TestResultClass implements ITestResult {
  id: string = "";
  studentTestId: string = "";
  testId: string = "";
  answerId: string = "";
  Comment: string = "";
  constructor(init?: TestResultClass) {
    Object.assign(this, init);
  }
}
